src/cljs/athens/electron.cljs

```diff
@@ -0,0 +1,89 @@
+(ns athens.electron
+  (:require
+    [athens.db :as db :refer [dsdb]]
+    [datascript.transit :as dt :refer [write-transit-str]]
+    [day8.re-frame.async-flow-fx]
+    [re-frame.core :refer [#_reg-event-db reg-event-fx inject-cofx reg-fx dispatch]]))
+
+
+(def electron (js/require "electron"))
+(def remote (.. electron -remote))
+
+(def dialog (.. remote -dialog))
+(def app (.. remote -app))
+
+
+(def fs (js/require "fs"))
+(def path (js/require "path"))
```

Comment:
This [post](https://clojureverse.org/t/guide-on-how-to-use-import-npm-modules-packages-in-clojurescript/2298) suggests that we can only do `(require :[]`) packages if we use ES6.

It also links to [webpack with cljs](https://github.com/clojure/clojurescript-site/blob/a197371d210df8ff6eca2c1e3703860fa11370c4/content/guides/webpack.adoc). Ugly common JS imports for now, sorry.